docs/enhancement-proposals/off-ep-0010.md

```diff
@@ -0,0 +1,134 @@
+# OFF-EP 0010 — Clarify ProperTorsion implementation of idivf and dihedral calculation
+
+**Status:** Submitted
+
+**Authors:** Lily Wang
+
+**Acceptance criteria:** Unanimity (4 approving reviews) or partial support (2 approvals and 2 week period with no reviews requesting changes)[https://openforcefield.atlassian.net/wiki/spaces/MEET/pages/2638774273/09-05-23+SMIRNOFF+Committee+Meeting]
+
+**Stakeholders:** John Chodera, David Mobley, Jeff Wagner, Lily Wang, Matt Thompson
+
+**Created:** 2024-02-07
+
+**Discussion:** [Issue #59](https://github.com/openforcefield/standards/issues/59), [Issue #60](https://github.com/openforcefield/standards/issues/60)
+
+**Implementation:** [``openforcefield/standards``](https://github.com/openforcefield/standards)
+
+## Abstract
+
+This change clarifies the implementation of SMIRKS matching, dihedral calculation, and `idivf="auto"` for ProperTorsion parameters.
+
+## Motivation and Scope
+
+A ProperTorsion is defined between a connected quartet of atoms i-j-k-l. The dihedral angle can be calculated using input vectors in two directions: ($r_{ij}$, $r_{jk}$, $r_{kl}$) or ($r_{ij}$, $r_{kl}$, $r_{kl}$), where $r_{ij} = x_{j} - x{i}$. It is currently unclear which standard to follow. While both directions yield the same result for symmetric torsions where the phase is 0 or pi, the choice of direction affects the sign for asymmetric torsions.
```

Comment:
I'm not sure what you mean by "The dihedral angle can be calculated using input vectors in two directions:" where you give triples of vectors `(a, b, c)`. Can you give an explicit mathematical expression for the dihedral angle `phi` in terms of these vectors `(a,b,c)`?

Also, the second `($r_{ij}$, $r_{kl}$, $r_{kl}$)` contains two repeated vectors.

Reply:
I've changed this section to a more qualitative description in words, and left the mathematical definition for the detailed description / specification itself.